### PR TITLE
Fixed Review Comments

### PR DESCRIPTION
**Pull request by Chamarie77**
Fixed Review Comments

### CODE REVIEW

File: MarsFramework/Test/TestStarts.cs [hunk 1]
```diff
@@ -23,10 +23,16 @@ public void Test_AddShareSkills()
                 ShareSkill page = new ShareSkill();
                 page.AddShareSkill(Driver);
```
Comment (amanbansalnz) [inline MarsFramework/Test/TestStarts.cs]: you should avoid passing the driver around, if you need it then create a variable in the page class and then use is inside the method, please fix this where ever you are passing the driver

File: MarsFramework/Pages/ProfilePage.cs [hunk 2]
```diff
@@ -1,37 +1,106 @@
...
 
+        //Click on Update Button
+        [FindsBy(How = How.XPath, Using = "(//input[@type = 'button'][@value = 'Update'])[1]")]
+        private IWebElement LanguageUpdateButton { get; set; }
+
+        //Click on Delete Icon
+        [FindsBy(How = How.XPath, Using = "(//i[@class = 'remove icon'])[1]")]
+        private IWebElement DeleteLanguage { get; set; }
+
+        private void GoToLanguageTab(IWebDriver Driver)
```
Comment (amanbansalnz) [inline MarsFramework/Pages/ProfilePage.cs]: avoid passing the driver, if you need this then make it available as a instance variable

**Review by amanbansalnz — CHANGES_REQUESTED** [review]
everything else looks good just fix the driver part

File: MarsFramework/Test/TestStarts.cs (hunk 1 above)
Comment (Chamarie77) [reply MarsFramework/Test/TestStarts.cs]: Thanks and fixed it.

File: MarsFramework/Pages/ProfilePage.cs (hunk 2 above)
Comment (Chamarie77) [reply MarsFramework/Pages/ProfilePage.cs]: Thanks & fixed it

**Review by amanbansalnz — APPROVED** [review]

**Pull request merged**